third_party/metrics_proto/OWNERS

```diff
@@ -0,0 +1,7 @@
+file://base/metrics/OWNERS
+
+per-file sampled_profile.proto=wittman@chromium.org
```

Comment:
Should I change these too?

Reply:
No, let's leave upstream OWNERs in place, we don't actually have an override in Cobalt for those ( ongoing discussion how and what we should be doing there )